src/index.html

```diff
@@ -5,10 +5,122 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>John Roberts' CV</title>
-    <link rel="stylesheet" href="./css/index.scss" />
-  </head>
-  <body>
+    <link rel="stylesheet" href="./css/index.scss"/>
+
+    <body>
     <div id="main"></div>
     <script src="./js/index.js"></script>
+      <nav> HOME </nav>
```

Comment:
Please put all the content within ` <div id="main"></div>`
`<script src="./js/index.js"></script>` needs to placed at the document bottom to execute.